src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/JsonSourceGeneratorTests.cs

```diff
@@ -699,10 +604,10 @@ public void UseUnderlyingTypeConverterForNullableType()
             byte[] referencedImage = CompilationHelper.CreateAssemblyImage(referencedCompilation);
 
             string source = """
-                using ReferencedAssembly;
```

Comment:
why this is no longer needed? how do we verify we can still reference

Reply:
It was never needed and the compiler was actually giving out a warning about this. It's just that the tests weren't always checking if the compilation did not produce any warnings or errors.